Directory.Build.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup>
     <MicrosoftPackageVersion>6.0.0</MicrosoftPackageVersion>
-    <MasaUtilsPackageVersion>0.5.0</MasaUtilsPackageVersion>
-    <MasaContribPackageVersion>0.5.0</MasaContribPackageVersion>
-    <MasaBlazorPackageVersion>0.5.0</MasaBlazorPackageVersion>
+    <MasaUtilsPackageVersion>1.0.0-rc.1</MasaUtilsPackageVersion>
```

Comment:
Utils and Contrib share the same version, which can be controlled by a parameter, and subsequent versions also use the same parameter to make it easier to understand